components/layout/Nav/AnchorLink.js

```diff
@@ -1,7 +1,9 @@
+import { Link } from 'react-router-dom';
+
 const AnchorLink = ({ href, navLinkClassName, children } = props) => (
-    <a href={href} className={navLinkClassName}>
+    <Link to={href} className={navLinkClassName}>
```

Comment:
While I was working on this I notice that the client side nav was broken and we were hitting the server everytime we browse, this is the fix to bring that back.